### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by amazingbrian**
**Patch description**
Just testing the GitHub desktop app because i'm tired of using VIM since we live in 2021 and should be able to use our mouse

**Testing steps**
Downloaded the app and went through the steps to make a PR 

**Other information**
Looks like it works

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 M+ bot for Silverash
 
 Leaderboard for M+ IO scores in guild 
+
+Testing Github Desktop XD
```
Comment (mus1cholic) [inline README.md:6]: Re-requesting changes so you can test that too

Also can't you add an actual description lmao

**Review by mus1cholic — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Ynoctis) [reply README.md:6]: I hope your test worked pog

**Review by Ynoctis — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (amazingbrian) [reply README.md:6]: Haha it worked! Thanks friends!

**Review by amazingbrian — COMMENTED** [review]
Close this please T_T

**Pull request closed without merging**